src/Coriolis/no_rotation.jl

```diff
@@ -1,6 +1,7 @@
 #####
 ##### Functions for non-rotating models
 #####
+@inline fᶠᶠᵃ(i, j, k, grid, ::Nothing) = zero(grid)
```

Comment:
```suggestion

@inline fᶠᶠᵃ(i, j, k, grid, ::Nothing) = zero(grid)
```

Reply:
this was not here before?? is it necessary?

Reply:
It is nice to have a fallback so that we can use it in parameterizations (e.g. qgleith needs an f)

Reply:
ok, I see... I was just wondering if this was leftover from some of your experiments (e.g., reconstructing f+ζ)